src/server/journal/journal_slice.cc

```diff
@@ -156,7 +156,7 @@ void JournalSlice::AddLogRecord(const Entry& entry, bool await) {
     FiberAtomicGuard fg;
     // GetTail gives a pointer to a new tail entry in the buffer, possibly overriding the last entry
     // if the buffer is full.
-    item = ring_buffer_->GetTail(true);
+    item = &dummy;
```

Comment:
removing the use of ring_buffer_ from replication?

Reply:
🤷🏻‍♂️ 

We seem to be okay with it https://github.com/dragonflydb/dragonfly/pull/2466